src/explore-education-statistics-frontend/src/modules/data-catalogue/components/Filters.tsx

```diff
@@ -116,6 +130,33 @@ export default function Filters({
             }}
           />
         </FormGroup>
+
+        {showClearFiltersButton && (
+          <ButtonText onClick={onClearFilters}>Clear filters</ButtonText>
+        )}
+
+        <FormRadioGroup
```

Comment:
As mentioned in another comment, we could use a `DataSetType` type here to constrain the option values and provide better type safety i.e.

```tsx
<FormRadioGroup<DataSetType> />
```